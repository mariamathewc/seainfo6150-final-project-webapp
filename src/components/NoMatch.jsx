import React from 'react';
import styles from "./NoMatch.module.css";

import Logo from "../images/error-404.png"

const NoMatch = (props) => {


	return (


		<div >
			<img src={Logo} alt="error" className={styles.error404} />
		</div>


	)

}

export default NoMatch 