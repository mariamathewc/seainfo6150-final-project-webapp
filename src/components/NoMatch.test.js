import React from "react";
import NoMatch from "./NoMatch.jsx";
import { render } from "@testing-library/react";
import { BrowserRouter } from 'react-router-dom';



describe("NoMatch test", () => {
	it("renders correctly", () => {
		const { container } = render(<BrowserRouter><NoMatch /></BrowserRouter>);
		expect(container).toMatchSnapshot();
	});
});