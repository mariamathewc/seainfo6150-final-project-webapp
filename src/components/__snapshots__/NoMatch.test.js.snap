// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch test renders correctly 1`] = `
<div>
  <div>
    <img
      alt="error"
      class="error404"
      src="error-404.png"
    />
  </div>
</div>
`;
